zipkin-storage/elasticsearch/src/main/java/zipkin/storage/elasticsearch/NativeClient.java

```diff
@@ -149,9 +149,9 @@
   }
 
   final TransportClient client;
-  final boolean flushOnWrites;
+  private final boolean flushOnWrites;
```

Comment:
I changed it back – as you say, not a big deal, IntelliJ just happened to mention that it could be a weaker visibility & so I made the warning go away :)